requirements.txt

```diff
@@ -1 +1 @@
-pydantic
+pandas==1.5.1
```

Comment:
If it is a library, which is potentially used in many project, wouldn't life be easier if we provide a range here? I mean, currently adding snack to project requirements is more likely to generate version conflicts. Snack doesn't use too much pandas so it would be relative easy to check which versions allows to do needed operations. What do you think?